ginivision/src/main/java/net/gini/android/vision/analysis/AnalysisFragmentInterface.java

```diff
@@ -11,17 +11,20 @@
 public interface AnalysisFragmentInterface {
     /**
      * <p>
-     *     You should call this method when you start the document analysis using the Gini API.
+     *     Call this method to hide the error shown before with
+     *     {@link AnalysisFragmentInterface#showError(String, String, View.OnClickListener)} or
+     *     {@link AnalysisFragmentInterface#showError(String, int)}.
      * </p>
      */
-    void startScanAnimation();
+    void hideError();
 
     /**
      * <p>
-     *     You should call this method when the document analysis has finished.
+     *     You should call this method after you've received the analysis results from the Gini
+     *     API and there have been no extractions found.
```

Comment:
What do you think about `You should call this method after you've received the analysis results from the Gini API without the required extractions.` ?